tests/devicetestcase_multipart.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: GPL-2.0-only
+
+#
+# Copyright (C) 2020 Elastio Software Inc.
```

Comment:
Should be 2022 unless this code was really written two years ago.

Reply:
Done